### PR TITLE
4963 - Hover Styles on Datagrid Listmode + Actions Button Appearance

### LINKED ISSUE #4963: Datagrid: Hover styles cut off in list style datagrid
**Description**
Hoover option element in a SohoDataGrid is cut off.

**To Reproduce** Steps to reproduce the behavior:
1. Make a datagrid example that has a isList: true and:
2. includes with a column that uses this meta data 
   
 ```javascript
     id: "open",
     name: "Actions",
     width: "5%",
     formatter: Soho.Formatters.Actions,
     icon: "more",
     headerTooltip: "Actions",
     resizable: false,
     menuId: "card-options",
     click: (e, a) => {
         this.onActionHandler(e, a);
     },
 ```

2. Hover over the option button, you will see that the colors looks cut off. 

**Expected behavior**
Hoover should not be cut off. 

**Version**
- ids-enterprise-ng: v4.37.0
- ids-enterprise-ng@9.2.0

**Screenshots** 
hoover

**Platform**
- Device: Desktop
- OS Version: Windows 10
- Browser Name: Chrome
- Browser Version: 88.0.4324.190 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by rob2d**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

When using a datagrid as a list and it has an "actions" row,  the hover effect on action buttons gave the visual impression of being cut off.
image

**Related github/jira issue (required)**:

Closes #4963.

**Steps necessary to review your pull request (required)**:

- pull branch `4963-hover-styles-on-datagrid-actions-btn`
- run the demo server and visit http://localhost:4000/components/datagrid/test--actions-reload-list.html
- Hover any cell in the actions column and the cell's background should not appear, only the menu tooltip
image

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [X] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 - `[Datagrid]` Fixed an issue where the filter menu would not repopen in some cases. ([#4995](https://github.com/infor-design/enterprise/issues/4995))
 - `[Datepicker]` Updated validation.js to check if date picker contains a time value ([#4888](https://github.com/infor-design/enterprise/issues/4888))
 - `[Datepicker]` Fixed a UI issue where the apply and cancel buttons were unable to see on small screens. ([#4950](https://github.com/infor-design/enterprise/issues/4950))
+- `[Datagrid]` Clean up hover appearance of data-grid actions button when the grid is viewed as a list. ([#4963](https://github.com/infor-design/enterprise/issues/4963))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:25]:
```suggestion
- `[Datagrid]` Clean up hover appearance of datagrid actions button when the grid is viewed as a list. ([#4963](https://github.com/infor-design/enterprise/issues/4963))
```

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Small fix...

a) i added an example page http://localhost:4000/components/datagrid/test--actions-reload-list.html for easier testing
b) instead of making the whole cell dark. can we just have the hover state on the button. its a little subtle but its already there in #0000 (plus cursor changes). This way you did it is just a bit inconsistent with how were were doing it elsewhere. 

Screen Shot 2021-04-01 at 9 31 27 AM

**Comment by rob2d** [comment]
@tmcconechy not sure I'm understanding, it seems that the functionality already works this way (before this branch)? (can see first screen from original issue)

**Comment by rob2d** [comment]
@tmcconechy thanks for clarification, made the change and updated PR instructions.

**Review by tmcconechy — APPROVED** [review]

**Review by chrisakinrinade — APPROVED** [review]

**Pull request merged**